### PR TITLE
add removal of data dumps...will add reinstatement in a sec

### PR DESCRIPTION
**Pull request by Calbonian**
**Type of change**

- [ ] Bugfix
- [ ] New Feature
- [ ] Existing Feature
- [X] Content Change

**Pre-review checklist**

- [ ] I have commented my code (where applicable)
- [ ] I have checked the spelling/grammar (where applicable)
- [ ] I have checked the dates and link references (where applicable)

**Brief description of the change:**

Adds the removal of data dumps event

### CODE REVIEW

File: timeline_data.json [hunk 1]
```diff
@@ -1,5 +1,18 @@
 {
   "items": [
+    {
+  "date_str": "2023-06-07",
+  "title": "Data Dumps Removed",
+  "slug": "data-dumps-removed",
+  "type": "licensing",
+  "summary": "Stack Exchange Inc. does not post the June 2023 data dump.",
```
Comment (samliew) [inline timeline_data.json]: `does` should be changed to `did`

File: timeline_data.json [hunk 2]
```diff
@@ -1,5 +1,18 @@
 {
   "items": [
+    {
+  "date_str": "2023-06-07",
+  "title": "Data Dumps Removed",
```
Comment (samliew) [inline timeline_data.json]: The previous data dumps weren't removed - only the scheduled script was disabled. The title needs rewording to make this clearer with the slug updated along with it.

File: timeline_data.json [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+  "slug": "data-dumps-removed",
+  "type": "licensing",
+  "summary": "Stack Exchange Inc. does not post the June 2023 data dump.",
+  "body": "<p>The job that uploads the data dump to Archive.org was disabled on 28 March, and marked to not be re-enabled without approval of senior leadership. This was mainly over concerns on AI using Stack Exchange data. This caused a large outrage and was eventually reverted on June 16, 2023 (9 days later).</p>",
+  "links": [
+    {
+      "text": "MSE Post",
+      "url": "https://meta.stackexchange.com/questions/389922/june-2023-data-dump-is-missing"
+    }
+  ],
```
Comment (samliew) [inline timeline_data.json]: There is a missing `},` after this line, causing the validation check to fail
https://github.com/samliew/se-timeline/actions/runs/5682375773/job/15401581241?pr=22

**Review by samliew — CHANGES_REQUESTED** [review]

**Comment by samliew** [comment]
Thanks for the contribution, I have added my review. If you need help with JSON, you can use the [Event Editor](https://se-timeline.glitch.me/event-editor) and submit an [Update Timeline issue](https://github.com/samliew/se-timeline/issues/new?assignees=samliew&labels=timeline&projects=&template=event_update.md) instead.

File: timeline_data.json (hunk 3 above)
Comment (Oaphi) [reply timeline_data.json]: Checked out the PR and fixed it — done!

File: timeline_data.json [hunk 4]
```diff
@@ -1,7 +1,23 @@
 {
   "items": [
     {
-      "classes": ["featured-event"],
+      "date_str": "2023-06-07",
+      "title": "June 2023 Data Dump Not Posted",
+      "slug": "data-dump-not-posted",
+      "type": "licensing",
+      "summary": "Stack Exchange Inc. did not post the June 2023 data dump.",
+      "body": "<p>The job that uploads the data dump to Archive.org was disabled on 28 March, and marked to not be re-enabled without approval of senior leadership. This was mainly over concerns on AI using Stack Exchange data. This caused a large outrage and was eventually reverted on June 16, 2023 (9 days later).</p>",
```
Comment (Oaphi) [inline timeline_data.json:9]: I'd go for a separate event for reinstatement with some additional information on how it transpired and remove the last sentence from here as a result.

**Review by Oaphi — DISMISSED** [review]

File: timeline_data.json (hunk 4 above)
Comment (samliew) [reply timeline_data.json:9]: Happy to merge as-is, until another PR comes in adding a resolution event, or further details about this event.

**Review by samliew — APPROVED** [review]

**Pull request merged**